doc/en/user/source/production/container.rst

```diff
@@ -54,6 +52,17 @@ For more information about JVM configuration, see the article `Performance tunin
     
    The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
    
+Enable the Marlin rasterizer
+----------------------------
+
+The Marlin rasterizer in Java 8 and getting better performance and scalability while rendering vector data in Java 8. 
+In order to enable it add the following among the JVM startup options::
+
+     -Xbootclasspath/a:$MARLIN_JAR 
+     -Dsun.java2d.renderer=org.marlin.pisces.MarlinRenderingEngine 
+
+where ``$MARLIN_JAR`` is the location of the ``marlin*.jar`` file located in the geoserver/WEB-INF/lib directory.
```

Comment:
In the start.sh I managed to make it version independent, I think it's important as Marlin gets new versions out with sometimes significant improvements. For example the latest has improvements for area filling, so it would be good to upgrade to it. On windows batch files getting the location might be trickier, but should be doable.